### DISCUSSION TITLE
How are people identifing their devices?

### DISCUSSION THREAD

**Discussion by adirburke** [mKeRix/room-assistant#1208] [Q&A]
Hi All,

I have heaps of bluetooth devices
how are you all getting their MAC addresses or identifying them in the config on RoomAssistant ?

I have a tile, and I have no idea how to get its MAC address for BlueTooth Classic

**Comment by kreene1987** [comment]
I use the device or the app to find the bluetooth address. You can also go far away from other bluetooth devices and just scan bluetooth on your phone and the address should pop up.
